ipn/localapi/localapi.go

```diff
@@ -202,7 +203,7 @@ func validHost(h string) bool {
 	if host == "localhost" {
 		return true
 	}
-	addr, err := netip.ParseAddr(h)
+	addr, err := netip.ParseAddr(host)
```

Comment:
whoa, what? this should be its own commit

Reply:
no good deed, etc etc

Reply:
https://github.com/tailscale/tailscale/pull/7114